sample/host.json

```diff
@@ -24,12 +24,19 @@
             "maxOutstandingRequests": 30
         },
         "queues": {
-            "visibilityTimeout": "00:00:10"
+            "visibilityTimeout": "00:00:10",
+            "maxDequeueCount": 3
         },
-        "eventHub": {
+        "eventHubs": {
```

Comment:
The section name was changed recently. See [here](https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubExtensionConfigProvider.cs#L20)